src/TConsole.cpp

```diff
@@ -80,7 +80,19 @@ TConsole::TConsole(Host* pH, const QString& name, const ConsoleType type, QWidge
 , mType(type)
 {
     auto quitShortcut = new QShortcut(this);
-    quitShortcut->setKey(Qt::CTRL | Qt::Key_W);
+#if defined(Q_OS_MAC)
```

Comment:
Ooops, I forgot about this MacOs part - as it was not included in my Windows/Linux test builds...